Code/Engine/GameEngine/Animation/Implementation/PropertyAnimComponent.cpp

```diff
@@ -443,10 +442,16 @@ void ezPropertyAnimComponent::ApplyAnimations(const ezTime& tDiff)
 
 ezTime ezPropertyAnimComponent::ComputeAnimationLookup(ezTime tDiff)
 {
+  m_AnimationRangeLow = ezMath::Clamp(m_AnimationRangeLow, ezTime::Zero(), m_AnimDesc->m_AnimationDuration);
```

Comment:
Clamp range here again since the range can be modified at runtime and the user doesn't know the actual duration